shopinvader/services/customer.py

```diff
@@ -193,3 +199,13 @@ def _prepare_create_response(self, binding):
             "role": binding.role,
         }
         return response
+
+    def _valid_before_binding(self, params):
```

Comment:
Maybe method name should be _validate_before_binding

Reply:
True, but I saw this function for another module.